nielsendtvr/src/main/java/com/segment/analytics/android/integrations/nielsendtvr/NielsenDTVRIntegration.java

```diff
@@ -0,0 +1,100 @@
+package com.segment.analytics.android.integrations.nielsendtvr;
+
+import com.segment.analytics.Properties;
+import com.segment.analytics.integrations.Integration;
+import com.segment.analytics.integrations.Logger;
+import com.segment.analytics.integrations.TrackPayload;
+import com.nielsen.app.sdk.AppSdk;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class NielsenDTVRIntegration extends Integration<AppSdk> {
+    public static final Factory FACTORY = new NielsenDTVRIntegrationFactory();
+
+    private final AppSdk appSdk;
+    private final Logger logger;
+
+    NielsenDTVRIntegration(AppSdk appSdk, Logger logger) {
+        this.appSdk = appSdk;
+        this.logger = logger;
+    }
+
+    @Override
+    public void track(TrackPayload trackPayload) {
+        String event = trackPayload.event();
+
+        switch (event) {
+            case "Video Content Started":
+                play(trackPayload);
+                loadMetadata(trackPayload);
+                sendID3(trackPayload);
+                break;
+            case "Video Playback Resumed":
+            case "Video Playback Seek Completed":
+            case "Video Playback Buffer Completed":
+                play(trackPayload);
+                sendID3(trackPayload);
+                break;
+            case "Video Playback Paused":
+            case "Video Playback Interrupted":
+            case "Video Content Completed":
+            case "Video Playback Buffer Started":
+            case "Video Playback Seek Started":
+                appSdk.stop();
+                break;
+            case "Video Playback Completed":
+                appSdk.end();
+                break;
+            case "Application Backgrounded":
+                appSdk.stop();
+        }
+    }
+
+    private void play(TrackPayload trackPayload) {
```

Comment:
Please add descriptions for each method (including private) using Javadoc format. Required fields are:
* `@param`
* `@thrown`
* `@return`

This will help a lot other contributors when the code starts getting complicated :)